indico/modules/events/abstracts/controllers/management.py

```diff
@@ -57,6 +57,18 @@ def _process(self):
         return RH._process(self)
 
 
+class RHManageAbstractBase(AbstractMixin, RHManageAbstractsBase):
+    """Base class for all abstract management RHs"""
+
+    def _checkParams(self, params):
+        RHManageAbstractsBase._checkParams(self, params)
+        AbstractMixin._checkParams(self)
+
+    def _checkProtection(self):
+        RHManageAbstractsBase._checkProtection(self)
+        AbstractMixin._checkProtection(self)
```

Comment:
It doesn't harm to leave it there in case we add some extra check in `AbstractMixin._checkProtection`.